pkgs/development/tools/continuous-integration/laminar/default.nix

```diff
@@ -50,25 +44,18 @@ in stdenv.mkDerivation rec {
     cp  ${js.vue-router}  js/vue-router.min.js
     cp  ${js.ansi_up}     js/ansi_up.js
     cp  ${js.Chart}       js/Chart.min.js
-    cp  ${css.bootstrap}  css/bootstrap.min.css
   '';
+
   postInstall = ''
-    mv $out/usr/share $out
-    mkdir $out/bin
-    mv $out/usr/{bin,sbin}/* $out/bin
-    rmdir $out/usr/{bin,sbin}
-    rmdir $out/usr
+    mv $out/usr/share/* $out/share/
+    rmdir $out/usr/share $out/usr
 
     mkdir -p $out/share/doc/laminar
     pandoc -s ../UserManual.md -o $out/share/doc/laminar/UserManual.html
-  '' + lib.optionalString (customConfig != null) ''
-    cp ${customConfig} /etc/etc/laminar.conf
-  '' + (if enableSystemd then ''
     sed -i "s,/etc/,$out/etc/," $out/lib/systemd/system/laminar.service
-    sed -i "s,/usr/sbin/,$out/bin/," $out/lib/systemd/system/laminar.service
-  '' else ''
-    rm -r $out/lib # it contains only systemd unit file
-  '');
+  '' + lib.optionalString (customConfig != null) ''
+    cp ${customConfig} $out/etc/laminar.conf
```

Comment:
I wonder if this is actually useful. Where does laminar look for configuration by default?

This feels like something that should be done in the NixOS modules instead.

Reply:
Well, laminar doesn‘t look into that file at all on its own.
That file contains environment variables supposed to be passed to laminar by systemd.
This option is useful when you want to use the vanilla systemd service that is part of this package.
Since I am planning on upstreaming a module for this, I don‘t think anyone would want to use it, though.

Reply:
I am okay with removal of "customConfig" option. Config is not part of laminar itself (unlike, e.g how dwm does it).